server.js

```diff
@@ -1,12 +1,40 @@
-const path = require("path")
-const express = require("express");
+const path = require('path');
+const express = require('express');
+const config = require('./config');
+const token = config.token;
+const apiUrl = 'https://app-hrsei-api.herokuapp.com/api/fec2/hr-rpp';
+const axios = require('axios');
+const _ = require('underscore');
 
 const app = express();
 const port = 3000;
-
-app.use(express.static(path.join(__dirname, "/client/dist")));
+app.use(express.json());
+app.use(express.urlencoded({ extended: true }));
+app.use(express.static(path.join(__dirname, '/client/dist')));
 // other configuration...
 
+
+app.post('/api/*', (req, res)=>{
+  console.log('body', req.body);
+
+  let options = req.body;
+  options.headers.Authorization = token;
+
+
+  axios(options).then((results)=>{
+    console.log('API Results:', results.data);
+    res.send(results.data);
```

Comment:
It would be better to send **statusCode** along with the data. Ex: **_res.status(statusCode).send(data)_**

Reply:
Will do for  this and below comment.